hydroflow_plus_example_flow/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+use hydroflow_plus::{q, quse, HydroflowContext, HydroflowNode, RuntimeData};
+
+fn filter_by_regex<'a, S: AsRef<str> + 'a>(
+    input: HydroflowNode<'a, String>,
+    pattern: RuntimeData<S>,
+) -> HydroflowNode<'a, String> {
+    quse!(::regex::Regex);
+
+    input.filter(q!({
+        let regex = Regex::new(pattern.as_ref()).unwrap();
+        move |x| regex.is_match(x)
+    }))
+}
+
+#[hydroflow_plus::flow(String)]
+pub fn my_example_flow<'a, S: AsRef<str> + 'a>(
+    ctx: &HydroflowContext<'a>,
```

Comment:
If there are no trait bounds on type parameters, the lifetime can usually be elided. This is a stress test to make sure that we enforce the same lifetime restrictions as the final runtime code.